opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/instrument.py

```diff
@@ -219,6 +220,24 @@ def __new__(cls, *args, **kwargs):
 
 
 class Histogram(_Synchronous, APIHistogram):
+    def __init__(
+        self,
+        name: str,
+        instrumentation_scope: InstrumentationScope,
+        measurement_consumer: "opentelemetry.sdk.metrics.MeasurementConsumer",
+        unit: str = "",
+        description: str = "",
+        advisory: Optional[MetricsInstrumentAdvisory] = None,
+    ):
+        super().__init__(
+            name,
+            unit=unit,
+            description=description,
+            instrumentation_scope=instrumentation_scope,
+            measurement_consumer=measurement_consumer,
+        )
+        self.advisory = advisory
```

Comment:
I think a follow up is to add [`attributes` advisory parameter](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.40.0/specification/metrics/api.md#instrument-advisory-parameter-attributes) to all instrument types. I think it's OK to just leave this as a one off in Histogram for now though